src/utils.ts

```diff
@@ -139,3 +139,17 @@ export function tsUnionOf(types: Array<string | number | boolean>): string {
   if (types.length === 1) return `${types[0]}`; // don’t add parentheses around one thing
   return `(${types.join(") | (")})`;
 }
+
+/**
+ * Determines if the http method being evaluated
+ * is permitted for the fetch request to the server
+ * to load the schema.
+ *
+ * @export
+ * @param {string} httpMethod
+ * @return {boolean} isValid
+ */
+export function isValidHTTPMethod(httpMethod: string | HTTPVerb): boolean {
```

Comment:
I found this function returns true for values like `123POST, POST123` etc. What about using [http.METHODS](https://nodejs.org/api/http.html#http_http_methods) to validate a value?

```suggestion
import {METHODS} from 'http'

export function isValidHTTPMethod(httpMethod: string | HTTPVerb): boolean {
  METHODS.includes(httpMethod.toUpperCase())
}
}
```

Reply:
Great point! I will fix this as well￼